aries-site/src/examples/templates/content-layouts/ResponsiveContentLayoutExample.js

```diff
@@ -0,0 +1,87 @@
+import React, { useContext } from 'react';
+import { Box, Grid, Header, Heading, ResponsiveContext } from 'grommet';
+import {
+  AppContainer,
+  PageContainer,
+  PageContainerContext,
+} from '../page-layouts/components';
+import { ContentArea } from '../page-layouts/anatomy/components';
+
+const parentGridColumns = {
+  xsmall: '100%',
+  small: '100%',
+  medium: ['auto', 'auto'],
+  large: ['auto', 'medium'],
+  xlarge: ['auto', 'medium'],
+};
```

Comment:
I'm thinking about the following structure for defining Grids. Specifically, defining properties which may want/need to be responsive and/or may differ from parent grid to nested children. In this example those are `columns` and `gap`; `areas` and `rows` may be possibilities for other instances. Proposed structure:
```
const myGrid = {
  columns: { ... },
  rows: { ... },
  areas: { ... },
  gap: { ... },
};
```

```suggestion
const parentGrid = {
  columns: {
    xsmall: '100%',
    small: '100%',
    medium: ['auto', 'auto'],
    large: ['auto', 'medium'],
    xlarge: ['auto', 'medium'],
  },
  gap: {
    xsmall: 'large',
    small: 'large',
    medium: 'medium',
    large: 'large',
    xlarge: 'large',
  },
};
```

☝️  this will also help with naming.

Reply:
Nice, I like that. I'll work on that.